src-library/MidiMessage.cpp

```diff
@@ -1163,6 +1163,10 @@ void MidiMessage::getSpelling(int& base7, int& accidental) {
 void MidiMessage::setMetaTempo(double tempo) {
    int microseconds = (int)(60.0 / tempo * 1000000.0 + 0.5);
    resize(6);
```

Comment:
Adding the setTempoMicrosections() function is great.

This line, `resize(6);`, should be moved into setTempoMicroseconds(); otherwise there will be problems if the message size is not set externally to 6 before calling setTempoMicrosections() directly outside of the setMetaTempo() function.  I will make that change after merging.